RampUpAuto/src/test/java/seleautomation/com/RampUpAuto/SearchOnGoogle.java

```diff
@@ -0,0 +1,28 @@
+package seleautomation.com.RampUpAuto;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SearchOnGoogle {
+    @Test
```

Comment:
As we are moving to use TestNG , let's benefit from the different annotations we have , as you can notice in the task we have like 3 parts of the testing , for the precondition and setting up your web driver you can use this annotation `@BeforeClass`